android/src/main/AndroidManifest.xml

```diff
@@ -14,4 +14,6 @@
     <uses-permission android:name="android.permission.MODIFY_AUDIO_SETTINGS"/>
     <uses-permission android:name="android.permission.ACCESS_NETWORK_STATE"/>
     <uses-permission android:name="android.permission.ACCESS_WIFI_STATE"/>
+    <uses-permission android:name="android.permission.BLUETOOTH_ADMIN"/>
+    <uses-permission android:name="android.permission.BLUETOOTH"/>
```

Comment:
Do we need those two? Even though it ended up failing in the app side, the main goal of https://github.com/senseobservationsystems/goalie-2-mobile-app/issues/5398 was to remove the requirement of those permissions, so Android 12 users wouldn't be requested for Bluetooth permissions in case they don't have bluetooth headphones connected when having the call, on? Or was it only due to the other permission `BLUETOOTH_CONNECT`?

Reply:
@jagamypriera Can you respond to this? Are these permissions still included upstream?

Reply:
those changes are from upstream. it won't show permission dialog to users